recipes/zstd/all/conanfile.py

```diff
@@ -90,3 +102,8 @@ def package_info(self):
         self.cpp_info.components["zstdlib"].libs = collect_libs(self)
         if self.settings.os in ["Linux", "FreeBSD"]:
             self.cpp_info.components["zstdlib"].system_libs.append("pthread")
+
+        if self.options.build_programs:
+            # TODO: Remove after dropping Conan 1.x from ConanCenterIndex
+            bindir = os.path.join(self.package_folder, "bin")
+            self.runenv_info.prepend_path("PATH", bindir)
```

Comment:
```suggestion
            self.env_info.PATH.append(bindir)
```
@klausholstjacobsen the `runenv_info` is Conan 2.x feature: https://docs.conan.io/2/reference/conanfile/methods/package_info.html#buildenv-info-runenv-info

For Conan 1.x , you should use `self.env_info.PATH` instead. I tested locally and this change works.